book/invalidation.md

```diff
@@ -162,7 +151,8 @@ class InputLayout(EmbedLayout):
             cmds.append(DrawCursor(self, self.font.measureText(text)))
 ```
 
-You can now edit the examples on this page in your browser---but
+You can now edit the examples on
+[this page](https://browser.engineering/invalidation.html) in your browser---but
```

Comment:
In other places the copyeditor suggests "the web version of this chapter". Did he not, here? Or you decided against?

Reply:
They recommended what I put here.